tests/integration-tests/tests/performance_tests/test_starccm.py

```diff
@@ -0,0 +1,95 @@
+
+
+def get_starccm_secrets(region_name):
+    secrets_manager_client = boto3.client("secretsmanager", region_name=region_name)
+    response = secrets_manager_client.get_secret_value(SecretId=STARCCM_LICENCE_SECRET)["SecretString"]
+    secrets = json.loads(response)
+    return secrets["podkey"], secrets["licpath"]
+
+
+def perf_test_difference(perf_test_result, number_of_nodes):
+    baseline_result = BASELINE_ELAPSED_TIME[number_of_nodes]
+    difference = perf_test_result-baseline_result
+    avg_result = (baseline_result+perf_test_result)/2
+    percentage_difference = abs(difference/avg_result)*100
+    percentage_difference *= -1 if difference < 0 else 1
+    return percentage_difference
+
+
+def starccm_installed(headnode):
+    cmd = "/shared/ec2-user/STAR-CCM+/16.02.008/STAR-CCM+16.02.008/star/bin/starccm+ --version"
+    try:
+        headnode.run_remote_command(cmd)
+        return True
+    except RemoteCommandExecutionError:
+        return False
+
+
+@pytest.mark.parametrize(
+    "number_of_nodes",
+    [[8, 16, 32]],
+)
+def test_starccm(
+    vpc_stack,
+    instance,
+    os,
+    region,
+    scheduler,
+    pcluster_config_reader,
+    clusters_factory,
+    number_of_nodes,
+    test_datadir,
+    scheduler_commands_factory,
+):
+    cluster_config = pcluster_config_reader(number_of_nodes=max(number_of_nodes))
+    cluster = clusters_factory(cluster_config)
+    logging.info("Cluster Created")
+    remote_command_executor = RemoteCommandExecutor(cluster)
+    if not starccm_installed(remote_command_executor):
+        logging.info("Installing StarCCM+")
+        remote_command_executor.run_remote_script(
+            str(test_datadir / "starccm.install.sh"), timeout=STARCCM_INSTALLATION_TIMEOUT, hide=False
+        )
+    logging.info("StarCCM+ Installed")
+    scheduler_commands = scheduler_commands_factory(remote_command_executor)
+    podkey, licpath = get_starccm_secrets(region)
+    performance_degradation = {}
+    for node in number_of_nodes:
+        num_of_tasks = node * TASK_VCPUS
+        command = f'sbatch --ntasks={num_of_tasks} starccm.slurm.sh "{podkey}" "{licpath}"'
+        result = remote_command_executor.run_remote_command(
+            [command], additional_files=[str(test_datadir / "starccm.slurm.sh")]
+        )
+        logging.info(f"Submitting StarCCM+ job with {node} nodes")
+        job_id = scheduler_commands.assert_job_submitted(result.stdout)
+        scheduler_commands.wait_job_completed(job_id, timeout=STARCCM_JOB_TIMEOUT)
+        scheduler_commands.assert_job_succeeded(job_id)
+        perf_test_result = remote_command_executor.run_remote_script((str(test_datadir / "starccm.results.sh")),
+                                                                     args=[job_id], hide=False)
+        logging.info(f"The elapsed time for {node} nodes is {perf_test_result.stdout}")
+        difference = perf_test_difference(float(perf_test_result.stdout), node)
+        logging.info(difference)
```

Comment:
This log line will be hard to understand because it will only contain a floating number. I suggest to add more context in the message, such as "Percentage difference for clyster size {nodes} between observed elapsed time ({observed_value}) and baseline ({baseline_value}): {percentage_difference}"